test/common.js

```diff
@@ -79,6 +80,19 @@ const helperFastForwardTime = async (seconds) => {
   ethers.provider.send('evm_mine');
 }
 
+const format6Dec = (amount) => {
+  return BigNumber.from(amount).div(sixDec);
+}
+
+const parse6Dec = (amount) => {
+  return BigNumber.from(amount).mul(sixDec);
+}
+
+const scaleUpForDecDiff = (reserve) => {
```

Comment:
can you generalise for any dec rather than just the delta between seuro and usdt?